TreeDB/docs/spec/typed-column-optimized-consumer-capabilities.md

```
@@ -110,21 +112,27 @@ Current logical value types covered: `bool`, `int64`, `float32`, `double`,
 (`uint8_vector`, `int8_vector`, `uint16_vector`, `int16_vector`,
 `uint32_vector`, `int32_vector`, `uint64_vector`, `int64_vector`,
 `float16_vector`, `bfloat16_vector`, `float32_vector`, `float64_vector`),
-`uint32_list`, `bytes`, and `adjacency_list`.
+#1931 fixed/packed code vectors (`byte_vector`, `packed_bit_vector`,
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)